pkg/landscaper/controllers/execution/controller.go

```diff
@@ -65,7 +65,14 @@ func (c *controller) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	}
 
 	if lsv1alpha1helper.IsCompletedExecutionPhase(exec.Status.Phase) {
-		logger.V(7).Info("execution item is already in completion phase")
+		op := execution.NewOperation(operation.NewOperation(logger, c.client, c.scheme, c.eventRecorder), exec, false)
+		phaseChanged, err := handleDeployItemPhaseChanges(ctx, op, logger, exec)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+		if !phaseChanged {
+			logger.V(7).Info("execution item is already in completion phase")
+		}
 		return reconcile.Result{}, nil
 	}
 
```

Comment:
Probably I do not understand the reason behind this pull request. Why do we only propagate phase changes but not e.g. changes of export data?

Reply:
hmm good point.
Currently no deployer is able to update the export without a configuration change.
But there are use cases in the future.
But I'm not sure we should add that in this PR. I think it includes lot more changes like e.g. tracking fo the previous data state and reading the export during every reconcile

Reply:
I agree, this could be interesting but we shouldn't add that in this PR.